### DISCUSSION TITLE
¿Descripción general del requerimiento?

### DISCUSSION THREAD

**Discussion by miguelapol** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#28] [Q&A]
¿Qué diferencias hay el nombre del proyecto a un Nombre de requerimiento?

**Comment by LeonardoAlvarezRivera** [comment]
El nombre del proyecto es el que se da en general, por ejemplo "Proyecto Abogabot" (este nombre de proyecto siempre será el mismo) y el nombre de requerimiento van a ser a la serie de características o acciones  que el cliente quiere en su sistema por ejemplo:

requerimiento 1:  Listado de clientes
requerimiento 2:  Demandas activas de un cliente
etc..

hasta que se describan todos los requerimientos del sistema

**Reply by miguelapol** [reply]
Muchas gracias
